src/server/generate/anime-genres.ts

```diff
@@ -0,0 +1,49 @@
+import fs from 'fs';
+import AnimeGenre from '../classes/animeGenre';
+import GenreObject from '../classes/genreObject';
+
+const getAnimeGenres = async () => {
+  try {
+    const tvData = fs.readFileSync('../static/anime_tv_batch.json');
+    const movieData = fs.readFileSync('../static/anime_movie_batch.json');
+    const data = tvData.toString().concat(movieData.toString());
+    const genres: AnimeGenre[] = [];
+    let genreCounter = 1;
+    const lines = data.toString().split('\n');
+    for (let i = 0; i < lines.length; i += 1) {
+      let line = lines[i];
+      if (line.includes('genres')) {
+        i += 1;
+        line = lines[i];
+        while (!line.includes(']')) {
+          if (line.includes('"name":')) {
```

Comment:
There should probably be a way to get a `value` directly from a `key` in a JSON object